sources/Valkey.Glide/Commands/IListCommands.cs

```diff
@@ -293,4 +356,298 @@ public interface IListCommands
     /// </example>
     /// </remarks>
     Task<ValkeyValue[]> ListRangeAsync(ValkeyKey key, long start = 0, long stop = -1, CommandFlags flags = CommandFlags.None);
+
+    /// <summary>
+    /// Returns the element at index <paramref name="index"/> in the list stored at <paramref name="key"/>.
+    /// The index is zero-based, so 0 means the first element, 1 the second element and so on.
```

Comment:
Maybe wrap numbers with `<c>` tag or use `langword` - for better docs readability